recipes/lmfit/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.9.3" %}
+
+package:
+  name: lmfit
+  version: {{ version }}
+
+source:
+  url: https://github.com/lmfit/lmfit-py/archive/{{ version }}.tar.gz
+  fn: lmfit-{{ version }}.tar.gz
+  sha256: e15c629b30cd70da525db6e66e3532d923a95e7a5f183906c83badf22c6be70c
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scipy
+
+  run:
+    - emcee
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - pandas
+    - python
+    - scipy
+
+test:
+  requires:
+    - nose
+  commands:
+    - nosetests -sv {{ environ.SRC_DIR }}
+  imports:
+    - lmfit
+
+about:
+  home: http://lmfit.github.io/lmfit-py/
+  license: BSD
```

Comment:
Maybe MIT? That's what they say [here]( https://lmfit.github.io/lmfit-py/ ). Though the language in that [`LICENSE`]( https://github.com/lmfit/lmfit-py/blob/c31906ec9e7a67d367c869f1d7f107f31dc0c123/LICENSE ) file doesn't look like MIT.